CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+
+## [1.1.0] - 2018-01-21
+### Changed
```

Comment:
As mentioned in the other PR, this should actually go under a `### Security` header rather than changed as it makes it clear to consumers that this update is important and should be prioritized over pulling in other types of changes.